### PR TITLE
[DRAFT] Fix wallet connections to the introduction.

### PR DESCRIPTION
**Pull request by fchirica**
Make wallet connect to the introducer only if a full node is not provided in config.

### CODE REVIEW

**Review by wong8885 — APPROVED** [review]

File: chia/wallet/wallet_node.py [hunk 1]
```diff
@@ -310,7 +310,8 @@ def set_server(self, server: ChiaServer):
             self.config["peer_connect_interval"],
             self.log,
         )
-        asyncio.create_task(self.wallet_peers.start())
+        if "full_node_peer" not in self.config:
```
Comment (mariano54) [inline chia/wallet/wallet_node.py:313]: We might have full node peer in config, but we are not actually running a full node. In that case we still need to start the node discovery.

File: chia/wallet/wallet_node.py [hunk 2]
```diff
@@ -320,7 +321,7 @@ async def on_connect(self, peer: WSChiaConnection):
             if peer.peer_node_id in peer_ids:
                 continue
             await peer.send_message(msg)
-        if not self.has_full_node() and self.wallet_peers is not None:
+        if "full_node_peer" not in self.config and not self.has_full_node() and self.wallet_peers is not None:
```
Comment (mariano54) [inline chia/wallet/wallet_node.py:324]: What is the purpose of this change?
